src/Propel/Generator/Command/DataDictionaryExportCommand.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * MIT License. This file is part of the Propel package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Propel\Generator\Command;
+
+use Propel\Generator\Manager\DataDictionaryExportManager;
+use Symfony\Component\Console\Exception\MissingInputException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Charles Crossan <crossan007@gmail.com>
+ */
+class DataDictionaryExportCommand extends AbstractCommand
+{
+    public const DEFAULT_OUTPUT_DIRECTORY = 'generated-datadictionary';
+
+    private const OPTION_OUTPUT_DIR = 'output-dir';
```

Comment:
Can we make those protected both?
Last thing from my side.

Reply:
ok